library/ui-strings/src/main/res/values/strings.xml

```diff
@@ -3287,4 +3286,31 @@
     <string name="onboarding_new_app_layout_feedback_message">Tap top right to see the option to feedback.</string>
     <string name="onboarding_new_app_layout_button_try">Try it out</string>
 
+    <string name="device_manager_filter_bottom_sheet_title">Filter</string>
+    <string name="device_manager_filter_option_all_sessions">All session</string>
+    <string name="device_manager_filter_option_verified">Verified</string>
+    <string name="device_manager_filter_option_verified_description">Ready for secure messaging</string>
+    <string name="device_manager_filter_option_unverified">Unverified</string>
+    <string name="device_manager_filter_option_unverified_description">Not ready for secure messaging</string>
+    <string name="device_manager_filter_option_inactive">Inactive</string>
+    <plurals name="device_manager_filter_option_inactive_description">
+        <item quantity="one">Inactive for %1$d day or longer</item>
+        <item quantity="other">Inactive for %1$d days or longer</item>
+    </plurals>
+    <string name="device_manager_other_sessions_title">Other sessions</string>
```

Comment:
For this string, I think we should reuse the existing `settings_sessions_other_title` string. Also I think we should rename the `settings_sessions` strings like in my draft PR: https://github.com/vector-im/element-android/pull/7110/files so that we can reuse it and see it is part of device manager.

Reply:
Done but I am not sure about renaming. Personally I prefer "settings_" prefix.

Reply:
I choosed to rename it since it is part of the device manager feature. Why do you prefer the `settings_` prefix?

Reply:
Oh you are right. I was checking wrong string. Done.